lib/twitter-ads/client.rb

```diff
@@ -3,7 +3,7 @@
 
 module TwitterAds
 
-  API_VERSION = '2'.freeze
+  API_VERSION = '3'.freeze
```

Comment:
this should be unnecessary given that i've made this change in a [previous PR](https://github.com/twitterdev/twitter-ruby-ads-sdk/pull/171/files#diff-159cb692d402ced7dfc6394a9a255caaR5)

Reply:
This is necessary as discussed offline